articles/css-in-js-solutions/index.md

```diff
@@ -0,0 +1,121 @@
+
+Well, apart from the cool feature of declaring media queries, here are some of the benefits of using Styletron:
+* Performance: To ensure high performance, Styletron follows the critical rendering concept by generating styles when needed by the demanding components. With this, browsers only load CSS rules that are currently in use rather than processing all CSS rules in a go.
+* Portability: Styletron ensures portability by ensuring declaration of styles written in JavaScript
+within your component, providing the same experience as using inline styles but all in CSS-in-JS solutions. 
+
+Styletron provides the above distinct advantages when used, the same advantages to other CSS-in-JS solutions also apply to it. You can get started with Styletron [here](https://www.styletron.org/getting-started).
+
+4. Emotion
+Like every other CSS-in-JS library, Emotion is a performant and flexible CSS-in-JS solution. Emotion makes use of String and Object styles to write CSS in JavaScript, and even better it ensures improved runtime performance by parsing styles with Babel and PostCSS.
+
+Emotion took into consideration approaches taken by other CSS-in-JS solutions like styled-components and glamor, so you tend to enjoy the same benefit as them. However, despite the foundation of existing solutions, Emotion was designed to solve the issue of specificity in CSS.
+
+Getting started with Emotion is also easy, you can follow this [guide](https://emotion.sh/docs/install) to start using Emotion in your project, and you can check out live code of Emotion Strings and Objects in declaring CSS styles in JavaScript [here](https://5bb1495273f2cf57a2cf39cc--emotion.netlify.com/).
+
+5. Material UI: Material UI is a React component that aims to ensure fast and easy web application development. The material UI is not only restricted to material UI components, but you can also employ the CSS-in-JS solution through the @material-ui/styles package you can find [here](https://www.npmjs.com/package/@material-ui/styles).
+
+Material UI has been using LESS and Inline CSS to style components, but that has come to an end since the modification to take full advantage of CSS-in-JS. Material UI also has similar advantages to styled-components. 
+
+In favor of performance, Material UI uses JSS as its core, which ensures speedy compilation from JS to CSS both at runtime and server-side. Material UI is small in size, occupying just 15kb gzipped space. Follow this guide to get started with the installation.
+
+### Conclusion
+CSS-in-JS changes the perspective of web application development from specificity to modularity, and towards implementing CSS-in-JS aptly, you now have a list of preceding CSS-in-JS solutions for web components examined in brief with links to get you started.
```

Comment:
**Flow edit:** modularity. You now have a list of modern CSS-in-JS solutions you can use in combination with web components with links to get you started.